package.json

```diff
@@ -13,6 +13,8 @@
     "url": "https://github.com/taskcluster/taskcluster-queue.git"
   },
   "dependencies": {
+    "ajv": "^4.2.0",
+    "ajv-keywords": "^1.1.1",
```

Comment:
hmm... we really should find out why or what is wrong here.. Adding dependencies like this isn't a solution as it'll spread to all other components and become unmaintainable... Maybe one of the upstream packages are missing a dependency...